Controllers/TodoController.cs

```diff
@@ -1,285 +1,207 @@
 using Microsoft.AspNetCore.Mvc;
-using Microsoft.EntityFrameworkCore;
-using TodoAPI.DAL.DBContexts;
-using TodoAPI.DAL.Entity;
 using TodoAPI.Models.RequestViewModels;
 using TodoAPI.Models.ResponseViewModels;
+using TodoAPI.Services.Interface;
 
 namespace TodoAPI.Controllers
 {
+    
+    [Route("todo")]
     [ApiController]
-    [Route("[controller]")]
     public class TodoController : ControllerBase
```

Comment:
Have you even tried building up the project before pushing the commits?
If Yes, then clone the same repo from dev branch and again build it.
It does not contain all the required files (repositories and services).